languages/README.md

```diff
@@ -0,0 +1,165 @@
+   and answering related issues.
+   Typically, reviewers of each language are members of a Github team which is named 
+   after the language code(i.e., zh-CN reviewers).
+
+### Translators
+
+A translator can start translating by following [the translation instructions](#how-to-translate).
+Translators should follow the [translation style guide](#translation-style-guide).
+
+### Reviewers
+
+Anyone, who builds [a new language structure](#how-to-add-a-new-language), will be the first reviewer
+of that language.
+
+Reviewers should track translation state of each document by editing 
+`languages/{language-code}/translations.md` file.
+
+There are 4 types of status:
+- Ready to start: any document is ready to be translated stay in this state.
+- Started: indicating someone is translating this document
+- PR reviewing: a translator finished the translation, and the work is being reviewed.
+- PR merged: the translation is successfully merged into the spec.
+
+Initially, the last edit time will be the time when the translation is merged.
+Reviewers should ensure translations are up-to-date with the English version as it changes, meanwhile 
+update the last edit time as well. 
+
+The following rules govern adding and removing reviewers:
+
+- Reviewers can be proposed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+- Reviewers can be removed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+## How to add a new language 
+
+This section is mainly for anyone who needs to build a new language structure.
+
+Fork [cloudevents/spec](https://github.com/cloudevents/spec/tree/main)
+to your Github repo. 
+
+Then check the [language list](languages.md) to see if the language
+you'd like to translate into already exists. If it doesn't exist, you can start to add a new one.
+
+### Step 1. build the structure
+
+Create a directory under `languages` directory, named after the specific 
+[language code](http://www.lingoes.net/en/translator/langcode.htm).
+
+Add a new item to the [language list](languages.md), which linking to the directory 
+you just created. 
+
+### Step 2. copy documents to right location
+
+Copy all original documents except `languages` directory to `languages/{language-code}`.
+There is no need to copy project-related files(i.e.,LICENSE, OWNERS).
+
+For example:
+```
+copy cloudevents/spec.md to languages/zh-CN/cloudevents/spec.md
+copy README.md to languages/zh-CN/README.md
+```
+
+Clear content of copied documents to indicate that the document is ready to be translated.
+
+Then copy the [reviewer list](zh-CN/reviewers.md) and the [translation list](zh-CN/translations.md)
+to `languages/{language-code}`.
+
+Check example structure [here](zh-CN).
+
+## How to publish translation tasks
+
+### Step 1. open a single issue referring to an untranslated document
+
+Create an issue with label "{language-code}-translation" to publish a translation assignment.
```

Comment:
I think only admins can create labels. We may need to limit people to creating issues and updating the first comment in the issue to indicate their status

Reply:
I think I can remove the whole Section `How to publish translation tasks`.
That seems like something admins should do rather than reviewers.
So that reviewers are only responsible for building a new language structure(only the first reviewer will do it), and reviewing.